src/home/search.vue

```diff
@@ -63,11 +61,10 @@
       input(placeholder="Search text")
 
       menu#search3.max
-        header.fixed
-          .field.large.prefix.suffix.no-margin
-            i.front(data-ui="#search3") arrow_back
-            input(placeholder="Search text")
-            i.front(data-ui="#search3") close
+        .field.large.prefix.suffix.no-margin.fixed
```

Comment:
Im facing some trouble here using this search. when i click and instantly type something the i cant delete the text typed from the first input

Reply:
Responded above 😁